### PR TITLE
Patch CSS FrontEnd

### PR DESCRIPTION
**Pull request by williamputraintan**
- Sticky footer is back
- Fix Consent PopUp (remove [Reactjs-popup](https://react-popup.elazizi.com/) and use [daisyUI-dropdown](https://daisyui.com/components/dropdown/) instead)

Screenshot 2023-05-16 at 19 45 14

### CODE REVIEW

File: application/frontend/src/index.css [hunk 1]
```diff
@@ -9,7 +9,7 @@ body {
 
 /* Major cards as displayed down the page */
 .card-major {
-  @apply card card-compact bg-base-100 shadow-lg;
+  @apply card card-compact static bg-base-100 shadow-lg;
```
Comment (andrewpatto) [inline application/frontend/src/index.css:12]: how does this alter the sticky footer?

Comment (andrewpatto) [reply application/frontend/src/index.css:12]: oh - wait - its the CtxProvider.. never mind..

File: application/frontend/src/providers/show-alert-provider.tsx [hunk 2]
```diff
@@ -28,32 +28,31 @@ export default function ShowAlert(props: { children: React.ReactNode }) {
...
+            <div className="container alert mt-4 shadow-2xl">
               <div>
-                <h3 className="font-bold">{alertProps.title}</h3>
-                <div className="text-xs">{alertProps.description}</div>
+                <div>
+                  <h3 className="font-bold">{alertProps.title}</h3>
+                  <div className="text-xs">{alertProps.description}</div>
+                </div>
+              </div>
+              <div className="flex-none">
```
Comment (andrewpatto) [inline application/frontend/src/providers/show-alert-provider.tsx]: can we daisy UI something here so it looks a little bit the same as the other alerts (I realise this one needs to sit over the top of the others so it isn't quite the same)

**Review by andrewpatto — APPROVED** [review]

File: application/frontend/src/providers/show-alert-provider.tsx (hunk 2 above)
Comment (williamputraintan) [reply application/frontend/src/providers/show-alert-provider.tsx]: Yup, this is the code from DaisyUI. Realising this, I changed it to re-use the alert component instead here.

File: application/frontend/src/index.css (hunk 1 above)
Comment (williamputraintan) [reply application/frontend/src/index.css:12]: Yes, this is for the popup component.

**Pull request merged**